patches/api/0419-Add-event-for-player-editing-sign.patch

```diff
@@ -0,0 +1,72 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: by77er <by77er@gmail.com>
+Date: Sat, 10 Jun 2023 19:06:24 -0400
+Subject: [PATCH] Add event for player editing sign
+
+
+diff --git a/src/main/java/com/destroystokyo/paper/event/player/PlayerEditSignEvent.java b/src/main/java/com/destroystokyo/paper/event/player/PlayerEditSignEvent.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..70b5773dc3bc66cd7f6be87eb5f490345b26935e
+--- /dev/null
++++ b/src/main/java/com/destroystokyo/paper/event/player/PlayerEditSignEvent.java
+@@ -0,0 +1,60 @@
++package com.destroystokyo.paper.event.player;
++
++import org.bukkit.block.Sign;
++import org.bukkit.block.sign.Side;
++import org.bukkit.entity.Player;
++import org.bukkit.event.Cancellable;
++import org.bukkit.event.HandlerList;
++import org.bukkit.event.player.PlayerEvent;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Called when a player edits a sign
++ */
++public class PlayerEditSignEvent extends PlayerEvent implements Cancellable {
++    private static final HandlerList handlers = new HandlerList();
++    private boolean cancel = false;
++    @NotNull private final Sign sign;
++    @NotNull private final Side side;
++
++    public PlayerEditSignEvent(@NotNull Player editor, @NotNull Sign sign, @NotNull Side side) {
++        super(editor);
++        this.sign = sign;
++        this.side = side;
++    }
++
++    /**
++     * Get the sign that was clicked
++     *
++     * @return {@link Sign} that was clicked
++     */
++    @NotNull
++    public Sign getSign() { return sign; }
```

Comment:
No need to make these methods in one line

Reply:
Expanded.